<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AttackMissionWidgetClass</class>
 <widget class="QMainWindow" name="AttackMissionWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>900</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>900</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AttackMissionWidget</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="SendBtn">
    <property name="geometry">
     <rect>
      <x>289</x>
      <y>560</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>不移动选择目标攻击下发</string>
    </property>
   </widget>
   <widget class="QComboBox" name="HumanBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>360</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>高度:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LatEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>320</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>240</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>当前位置</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="LonEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>280</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>160</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>无人机:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AltEdit">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>360</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>40</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>执行者:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>经度:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ExecuterEdit">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>100</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>人员:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="UavBox">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>160</y>
      <width>150</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>320</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>纬度:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="TargetListWidget">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>650</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>30</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>目标列表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SendBtn_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>600</y>
      <width>191</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>不移动攻击所有目标下发</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
